ert_logging/logger.conf

```diff
@@ -34,6 +34,10 @@ loggers:
     level: DEBUG
     handlers: [eefile]
     propagate: no
+  ert_shared.ensemble_evaluator.prefect_ensemble:
+    level: ERROR
```

Comment:
At this point in development (i.e. most of the evaluator is behind a feature flag) I'm fine with `DEBUG`.